doc/api/next_api_changes/2019-08-22-DS.rst

```diff
@@ -0,0 +1,5 @@
+``linthresh`` paramter to `LogFormatter` is deprecated
```

Comment:
I think all these will need a dot in front of the class name to keep the reference resolver happy? we'll see the docs build...

Reply:
Parameters should be `*linthresh*` too (also note typo in paramter).